scout/server/blueprints/variants/templates/variants/components.html

```diff
@@ -82,8 +82,20 @@
       <div>
         <strong>Local (arch.)</strong>: {{ variant.local_obs_old or 0 }} obs.
       </div>
+      {% if variant.cosmic_ids %}
+      <strong>CosmicIDs</strong>:
+      <div>
+        {% for cosmicID in variant.cosmic_ids %}
+        cosm{{cosmicID}}
+        {% endfor %}
+      </div>
+      {% endif %}
     </div>
   ">
     {{ variant.gnomad_frequency|human_decimal if variant.gnomad_frequency else '~' }}
   </div>
+  {% if variant.cosmic_ids %}
+  <a href="{{ variant.cosmic_link }}" class="badge badge-light text=white visited =white" target="_blank">Cosmic</a>
```

Comment:
Is it possible to post an image how this looks like?

Reply:
![Screenshot 2020-04-08 at 08 49 52](https://user-images.githubusercontent.com/1150065/78753414-08440a00-7976-11ea-911e-ac716e946879.png)